action.py

```diff
@@ -58,6 +151,14 @@ def _fatal_help(msg):
 if entrypoint:
     sigstore_conformance_args.extend(["--entrypoint", entrypoint])
 
+gh_token = os.getenv("GHA_SIGSTORE_CONFORMANCE_GITHUB_TOKEN")
```

Comment:
Same comment as https://github.com/sigstore/sigstore-conformance/pull/74/files#r1177391839 -- maybe this?

```suggestion
gh_token = os.getenv("GITHUB_TOKEN")
```

Reply:
[79b2b1e](https://github.com/sigstore/sigstore-conformance/pull/74/commits/79b2b1e721b310dfe2bc4f0dc1b5e4001cf2efc3)